packages/react-component-library/src/components/DataGrid/DataGrid.stories.tsx

```diff
@@ -19,6 +22,7 @@ const generateRandomData = (length: number): Order[] => {
       id: i + 1,
       productName: faker.commerce.productName(),
       quantity: faker.number.int({ min: 1, max: 100 }),
+      price: `£${faker.commerce.price()}`,
```

Comment:
can we seed the random number generator so that it always returns the same "random" numbers. Then we won't fail visual regression every time?

Reply:
Ah yes, I forgot about this. I will have to strip faker out I think. All good i'll just take a static copy.